internal/host/installcmd.go

```diff
@@ -191,6 +191,11 @@ func getBootDevice(log logrus.FieldLogger, hwValidator hardware.Validator, host
 		return host.InstallationDiskPath, nil
 	}
 
+	// TODO: We generally shouldn't reach this point on any version containing this comment.
```

Comment:
I think you can remove this remark and also remove lines: 199-205, just add the following: if host.InstallationDiskPath == "" { 
		err := errors.Errorf("Failed to get valid disks on host with id %s", host.ID)
		log.Errorf("Failed to get valid disks on host with id %s", host.ID)
		return "", err
} 
It will have the same behavior as before and you will have only one method that handles the installation disk

Reply:
The way I see it what you suggest may cause potential errors during the rollout of the new version - if inventory updates reach the older versions, there won't be an installation disk path. Then if the installation command reaches the new version - I want to make sure it knows to re-evaluate the installation disk path instead of assuming `UpdateInventory` took care of that.

Reply:
ok, now it's clear. 
Maybe we should add a task for future release to remove this code

Reply:
https://issues.redhat.com/browse/MGMT-3509